core-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java

```diff
@@ -156,6 +159,7 @@ protected JComponent createCenterPanel() {
   @Override
   protected void doOKAction() {
     if (getOKAction().isEnabled()) {
```

Comment:
Well, I think initially it can, at least until the validator kicks in. We can explore this in another bug though.